### PR TITLE
Add note about Raspberry Pi OS 32 vs 64-bit and support

### PR DESCRIPTION
**Pull request by hfiguiere**
Make it clear what are the expectations for Raspberry Pi.

If you have suggestion on the copy let me know.

Also I haven't been able to `bundle install` the page generator so I haven't been able to review the output yet.

### CODE REVIEW

File: data/distro.yml [hunk 1]
```diff
@@ -405,6 +405,7 @@
         <pre><code>
           <span class="unselectable">#</span> flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
         </code></pre>
+        <p><b>Important note:</b> As of March 2021, Raspberry Pi computer still ship with the 32-bit version of Raspberry Pi OS. However Flathub started phasing out support for that architecture. If you consider Flathub as an important source of applications, it is recommended to use Raspberry Pi OS 64-bit as newer application are more likely to available, and other one updated.</p>
```
Comment (AsciiWolf) [inline data/distro.yml]: "as newer application are more likely to available, and other one updated." - This does not sound right to me.

Comment (hfiguiere) [reply data/distro.yml]: yeah missing words. Will fix. 

(and I re-read twice before submitting)

File: data/distro.yml [hunk 2]
```diff
@@ -405,6 +405,7 @@
         <pre><code>
           <span class="unselectable">#</span> flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
         </code></pre>
+        <p><b>Important note:</b> As of March 2021, Raspberry Pi computers still ship with the 32-bit version of Raspberry Pi OS. However Flathub started phasing out support for that architecture. If you consider Flathub as an important source of applications, it is recommended to use Raspberry Pi OS 64-bit as newer applications are more likely to be available for that platform, while other one more likely to be updated.</p>
```
Comment (AsciiWolf) [inline data/distro.yml]: "while other one more likely to be updated" - this part still does not seem to be correct to me.

Comment (hfiguiere) [reply data/distro.yml]: I just removed it.

**Comment by mwleeds** [comment]
Thanks, I'll assume you have the info correct on this

**Pull request merged**